scapy/contrib/rpl_metrics.py

```diff
@@ -0,0 +1,284 @@
+# This file is part of Scapy.
+# See http://www.secdev.org/projects/scapy for more information.
+#
+# Scapy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# Scapy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Scapy.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Copyright (C) 2020 Rahul Jadhav <nyrahul@gmail.com>
+
+# RFC 6551
+# scapy.contrib.description = Routing Metrics used for Path Calc in LLNs
+# scapy.contrib.status = loads
+
+"""
+RFC 6551 - Routing Metrics Used for Path Calculation in LLNs
+"""
+
+import struct
+from scapy.compat import orb
+from scapy.packet import Packet, bind_layers
+from scapy.fields import ByteEnumField, ByteField, ShortField, BitField, \
+    BitEnumField, FieldLenField, StrLenField, IntField
+from scapy.layers.inet6 import _PhantomAutoPadField, _OptionsField
+from scapy.contrib.rpl import rploptsstr, DIS, DIO, DAO, DAOACK, DCO, DCOACK
```

Comment:
`rploptsstr` isn't defined. This import fails

Reply:
Sorry, mistake. Committed in haste. Fixed.